doc/conf.py

```diff
@@ -0,0 +1,293 @@
+# -*- coding: utf-8 -*-
+#
+# conf.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Readthedocs configuration file
+------------------------------
+
+Do not work with local sphinx.
```

Comment:
What precisely do you mean by this? Is it an instruction to the user?

Reply:
The unimportant note is now deleted.